Search/FindKPairsWithSmallestSums.swift

```diff
@@ -0,0 +1,25 @@
+func kSmallestPairs(_ nums1: [Int], _ nums2: [Int], _ k: Int) -> [[Int]] {
```

Comment:
Could you add the question link and your idea to solve the question? You could reference other swift files for details.